packages/babel-jest/src/__tests__/index.ts

```diff
@@ -49,15 +54,18 @@ test('Returns source string with inline maps when no transformOptions is passed'
       configString: JSON.stringify(makeProjectConfig()),
       instrument: false,
       transformerConfig: {},
-    },
-  ) as any;
+    } as TransformOptions<BabelTransformOptions>,
+  );
+
   expect(typeof result).toBe('object');
   expect(result.code).toBeDefined();
   expect(result.map).toBeDefined();
   expect(result.code).toMatch('//# sourceMappingURL');
   expect(result.code).toMatch('customMultiply');
-  expect(result.map!.sources).toEqual(['dummy_path.js']);
-  expect(JSON.stringify(result.map!.sourcesContent)).toMatch('customMultiply');
+  expect((result as BabelFileResult).map!.sources).toEqual(['dummy_path.js']);
+  expect(
+    JSON.stringify((result as BabelFileResult).map!.sourcesContent),
+  ).toMatch('customMultiply');
```

Comment:
Ah.. `BabelFileResult` is maybe even better since this is `babel-jest` test. The value and type comes from Babel:

https://github.com/facebook/jest/blob/eafabf99e64ac03f0af40ca26f24ebbb0f401c77/packages/babel-jest/src/index.ts#L240-L247

Reply:
Perfect 👍